config/jobs/kubernetes/wg-k8s-infra/trusted/wg-k8s-infra-trusted.yaml

```
@@ -22,6 +22,80 @@ periodics:
       args:
       - -c
       - "cd groups && make run -- --confirm"
+- name: ci-k8sio-audit
+  interval: 6h
+  cluster: k8s-infra-prow-build-trusted
+  decorate: true
+  max_concurrency: 1
+  annotations:
+    testgrid-dashboards: wg-k8s-infra-k8sio
+    testgrid-alert-email: hh@ii.coop
+    testgrid-num-failures-to-alert: '100'
+  extra_refs:
+  - org: kubernetes
+    repo: k8s.io
+    base_ref: master
+  - org: kubernetes
+    repo: test-infra
+    base_ref: master
+  spec:
```

We should consider a github team to use for this instead, but for now we want to make sure you can re-run this for troubleshooting
```suggestion
  rerun_auth_config:
    github_users:
      - hh
  spec:
```

---

How would I go about triggering a rerun for a periodic job?